StitchCoreServicesMongoDbRemote/Sources/StitchCoreServicesMongoDbRemote/RemoteFindOptions.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+import MongoSwift
+
+/// Options to use when executing a `find` command on a `RemoteMongoCollection`.
+public struct RemoteFindOptions {
+    /// The maximum number of documents to return.
+    public let limit: Int64?
+    
+    /// Limits the fields to return for all matching documents.
+    public let projection: Document?
+    
+    /// The order in which to return matching documents.
+    public let sort: Document?
+    
+    /// Convenience initializer allowing any/all parameters to be optional
+    public init(limit: Int64? = nil, projection: Document? = nil, sort: Document? = nil) {
+        self.limit = limit
+        self.projection = projection
+        self.sort = sort
+    }
+    
+    // Encode everything
+    internal enum CodingKeys: String, CodingKey {
```

Comment:
I think I'll go with the former approach of moving the key enums to the Collection class, and naming them something else other than `CodingKeys`. I'm hesitant to go with the latter approach since that would deviate further from the driver